packages/components/select-v2/src/useSelect.ts

```diff
@@ -105,7 +105,7 @@ const useSelect = (props: ISelectV2Props, emit) => {
     afterComposition: (e) => onInput(e),
   })
 
-  const { wrapperRef, isFocused } = useFocusController(inputRef, {
+  const { wrapperRef, isFocused, handleBlur } = useFocusController(inputRef, {
```

Comment:
Both `handleClickOutside` in this file and `v-click-outside:[popperRef]="handleClickOutside"` in the `select.vue` file can be removed.

Reply:
link #17691